cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/batch/dataset/MultipleOutputs.java

```diff
@@ -52,6 +52,8 @@
 public class MultipleOutputs implements Closeable {
 
   private static final String MULTIPLE_OUTPUTS = "hconf.mapreduce.multipleoutputs";
+  private static final String PREFIXED_CONF_PREFIX = "hconf.namedConf.";
```

Comment:
nit: I believe we typically don't use camelCase in property names, but instead we use dot notation.